ios/Podfile

```diff
@@ -43,11 +43,7 @@ setup_permissions(
   ]
 )
 
-linkage = ENV.fetch( "USE_FRAMEWORKS", nil )
```

Comment:
This is the context for this, right? https://github.com/reactwg/react-native-new-architecture/discussions/115

LGTM!